bookwyrm/activitypub/__init__.py

```diff
@@ -34,3 +36,10 @@
 def parse(activity_json):
     """figure out what activity this is and parse it"""
     return naive_parse(activity_objects, activity_json)
+
+
+# pylint: disable=unnecessary-pass
+class UserIsGoneError(HTTPError):
+    """error class for when a user is banned or deleted"""
```

Comment:
(Optional suggestion.)

In the current form of the PR, this exception is only used inside inbox.py.

It would be cleaner, I think, to define it in that file (and it need not inherit from HTTPError, just `Exception` would be fine).

Additionally, in the end the exception has nothing to do with the 410 Gone error (just locally deleted users), so it may as well be called something like `UserIsDeleted`.